### PR TITLE
Added extract_custom_artifacts function in utils.py

### PR DESCRIPTION
**Pull request by roshanmaskey**
Created a new function extract_custom_artifacts that extracts artifacts based on a yaml file.

### CODE REVIEW

File: turbinia/lib/utils.py [hunk 1]
```diff
@@ -97,6 +97,43 @@ def extract_artifacts(artifact_names, disk_path, output_dir, credentials=[]):
   return _image_export(image_export_cmd, output_dir)
 
 
+def extract_custom_artifacts(artifact_names, artifact_definition_file,
```
Comment (aarontp) [inline turbinia/lib/utils.py:100]: This seems to be basically the same method as `extract_artifacts` with one extra parameter.  Should we just add that parameter as an optional parameter to the other method to reduce some code duplication?

File: turbinia/lib/utils.py [hunk 2]
```diff
@@ -97,6 +97,43 @@ def extract_artifacts(artifact_names, disk_path, output_dir, credentials=[]):
...
+                             disk_path, output_dir, credentials=[]):
```
Comment (aarontp) [inline turbinia/lib/utils.py:101]: Looks like `yapf` wants this with a newline after the method name and open parenthesis (that's why one of the tests is failing).

**Review by aarontp — COMMENTED** [review]
Cool, just one comment about possibly removing some code duplication. 

Maybe in a future PR we can migrate the binary_extractor task to use this: (https://github.com/google/turbinia/blob/master/turbinia/workers/binary_extractor.py#L107).

**Comment by roshanmaskey** [comment]
`[database_servers.yaml](https://github.com/ForensicArtifacts/artifacts/blob/main/data/database_servers.yaml)` is available through ForensicArtifacts.

This workaround is no longer required. Closing this pull request.

**Pull request closed without merging**